examples/cordova-sample/MyApp/www/js/index.js

```diff
@@ -102,8 +71,256 @@ const app = {
+      }
+    );
+  },
+
+  loginKnown: function() {
+    Purchases.logIn(
+      "test",
+      info => {
+        setStatusLabelText(info);
+      },
+      error => {
+        setStatusLabelText(error);
+      }
+    );
+  },
+
+  logOut: function() {
+    Purchases.logOut(
+      info => {
+        setStatusLabelText(info);
+      },
+      error => {
+        setStatusLabelText(error);
+      }
+    );
+  },
+
+  syncPurchases: function() {
+    Purchases.syncPurchases();
+    setStatusLabelText("Sync Purchases called. This method does not have a callback.");
+  },
+
+  restorePurchases: function() {
+    Purchases.restorePurchases(
+      info => {
+        setStatusLabelText(info);
+      },
+      error => {
+        setStatusLabelText(error);
+      }
+    )
+  },
+
+  canMakePayments: function() {
+    Purchases.canMakePayments(
+      info => {
+        setStatusLabelText(info);
+      },
+      error => {
+        setStatusLabelText(error);
+      }
+    )
+  },
+
+  setSubsAttributes: function() {
+    Purchases.setPhoneNumber("12345678");
+    Purchases.setDisplayName("Garfield");
+    Purchases.setAttributes({ "favorite_cat": "garfield" });
+    Purchases.setEmail("garfield@revenuecat.com");  
+    Purchases.setAdjustID("AdjustID");
+    Purchases.setAppsflyerID("AppsflyerID");
+    Purchases.setFBAnonymousID("FBAnonymousID");
+    Purchases.setMparticleID("MparticleID");
+    Purchases.setOnesignalID("OnesignalID");
+    Purchases.setAirshipChannelID("AirshipChannelID");
+    Purchases.setFirebaseAppInstanceID("FirebaseAppInstanceID");
+    Purchases.setMixpanelDistinctID("MixpanelDistinctID");
+    Purchases.setCleverTapID("CleverTapID");
+    Purchases.setMediaSource("MediaSource");
+    Purchases.setCampaign("Campaign");
+    Purchases.setAdGroup("AdGroup");
+    Purchases.setAd("Ad");
+    Purchases.setKeyword("Keyword");
+    Purchases.setCreative("Creative");
+    Purchases.collectDeviceIdentifiers();
+    setStatusLabelText("subscriber attributes set");
+  },
+
+  checkIntroEligibility: function() {
+    Purchases.getOfferings(
+      offerings => {
+        var productIdentifiers = [];
+        for (const [key, offering] of Object.entries(offerings.all)) {
+          offering.availablePackages.forEach(package => {
+            productIdentifiers.push(package.product.identifier);
+          });
+        }
+        
+        Purchases.checkTrialOrIntroductoryPriceEligibility(
+          productIdentifiers,
+          info => {
+            setStatusLabelText(info);
+          }
+        );
+      },
+      error => {
+        setStatusLabelText(error);
+      }
+    );
+  },
+  
+  isAnonymous: function() { 
+    Purchases.isAnonymous(
+      isAnonymous => {
+        setStatusLabelText(isAnonymous);
+      }
+    );
+  },
+  
+  invalidateCustomerInfoCache: function() {
+    setStatusLabelText("invalidating customer info cache");
+    Purchases.invalidateCustomerInfoCache();
+  },
+  
+  simulatesAskToBuyInSandbox: false,
```

Comment:
I think it is a bit strange how we setup variables and functions as part of this map... Probably an opportunity for cleanup, but I think getting it working is more important right now 👍